### ISSUE TITLE
Nginx proxy pass

### ISSUE THREAD

**Issue by ovidiubica** [evandrofisico/roundcube-imageproxy#1]
I'm wondering if you configured this plugin on Nginx as I cannot get it to work.

**Comment by evandrofisico** [comment]
Yes, I'm currently using the plugin. In my current configuration, both nginx and imageproxy are running on the same machine, with multiple instances of roundcube running in other hosts.

 Is imageproxy running? Can you try to to test it without using the signature key to see if it actually retrieves images?

**Comment by ovidiubica** [comment]
I tried it without the security key. It appears that I'm stuck on NginX rewriting the URL, as it merges the double slash.
`2019/10/15 21:27:09 invalid request URL: malformed URL "/api/imageproxy/sCMU_NQwPExOKOnqXculZ4c7ZJkWTyV3Y8e2TfHW8ekY=/http:/static.contactlab.it/custom_spa/2015/20150218/twitter.png": must provide absolute remote URL`

I tried this [Nginx location](https://github.com/willnorris/imageproxy/tree/reverseproxy#nginx) without success.

**Comment by loxK** [comment]
It is an [imageproxy](https://github.com/willnorris/imageproxy/issues) issue, has nothing to do with this roundcube plugin. I have it running on nginx, don't forget the trailing slash:

```
location /imageproxy/ {
    proxy_pass http://localhost:8080/;
}
```
